src/components/molecules/PublishedDate/index.tsx

```diff
@@ -11,7 +12,7 @@ export default function PublishedDate({ date }: Props) {
 
   return (
     <Box display="flex" gap={2}>
-      <Image src="/clock.svg" alt="" width={16} height={16} priority />
+      <TimeIcon mt={1} />
```

Comment:
clock.svg cannot be seen when the page is on dark mode, so I replaced it with TimeIcon.